public/partials/membership_card.html

```diff
@@ -18,7 +18,7 @@ <h3 class="white">Membership</h3>
         <div flex layout="row">
             <md-icon class="no-margin" md-svg-icon="/images/icons/icons_farmer_active.svg"></md-icon>
             <div layout-align="center" style="text-align: center;" layout="column" ng-class="{'md-offset': 1}">
-                Active
+                {{isActive(memberships[0].mi_expiration.split("T")[0])?  "Active" :  'Inactive'}}
```

Comment:
@paradoxxjm had mentioned that displaying only the most recent membership info is needed... the others... not so much.  Confirm with him @tremainebuchanan and @nick22891